apps/llm-client/src/components/settings/endpoint-dialog.tsx

```
@@ -0,0 +1,217 @@
+            {blocking ? "Can't reach your llama-server" : "Server endpoint"}
+          </DialogTitle>
+          <DialogDescription>
+            {blocking
+              ? "llm-client needs a reachable OpenAI-compatible server before it can start. Enter an address below and verify it."
+              : "Point llm-client at any OpenAI-compatible server. The endpoint is verified before it's saved."}
+          </DialogDescription>
+        </DialogHeader>
+        <div className="flex flex-col gap-2">
+          <label className="text-xs font-medium text-muted-foreground">
+            Base URL
+          </label>
```

Biome (2.4.10)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)